CHANGELOG.md

```diff
@@ -6,6 +6,24 @@ All notable changes to secrets-store-csi-driver-provider-gcp will be documented
 
 None
 
+## v1.5.0
+
+Images:
+
+* `asia-docker.pkg.dev/secretmanager-csi/secrets-store-csi-driver-provider-gcp/plugin:v1.5.0`
+* `europe-docker.pkg.dev/secretmanager-csi/secrets-store-csi-driver-provider-gcp/plugin:v1.5.0`
+* `us-docker.pkg.dev/secretmanager-csi/secrets-store-csi-driver-provider-gcp/plugin:v1.5.0`
+
+Digest: `sha256:a493a78bbb4ebce5f5de15acdccc6f4d19486eae9aa4fa529bb60ac112dd6650`
+
+
+### Changes & Security Fixes
+
+* Dependencies upgrade
+* Change the owner of the mount volume to the user other than root. [#370](https://github.com/GoogleCloudPlatform/secrets-store-csi-driver-provider-gcp/pull/370)
+* Removing and syncing duplicate configs [#349](https://github.com/GoogleCloudPlatform/secrets-store-csi-driver-provider-gcp/pull/349)
+
```

Comment:
IIRC we have also added a custom service agent flag. https://github.com/GoogleCloudPlatform/secrets-store-csi-driver-provider-gcp/pull/354

Reply:
This change mostly intended for internal use thus not included in changelog. Just like we didn't include other configurable variable changes in past releases e.g. 1.4.0: https://github.com/GoogleCloudPlatform/secrets-store-csi-driver-provider-gcp/pull/347/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed

https://github.com/GoogleCloudPlatform/secrets-store-csi-driver-provider-gcp/pull/322/files